lib/interactor/organizer.rb

```diff
@@ -76,7 +76,8 @@ module InstanceMethods
       # Returns nothing.
       def call
         self.class.organized.each do |interactor|
-          interactor.call!(context)
+          self.send(interactor) if interactor.is_a?(Symbol)
```

Comment:
Seems to me we have avoid additional nesting at the cost of logic duplication. Is it fine?